top.wake

```diff
@@ -47,19 +47,48 @@ global def bitstreamTarget board dut buildRoot =
 
 global def simTarget dut buildRoot simName waves =
   def dutTests = subscribe dutTests | filter (_.getDUTTestFilter dut)
-  map (singleSimTarget simName waves dut buildRoot) dutTests
-  | zip dutTests
-  | map getDUTTestName.editPairFirst
-  | aggregateTestResults
-  | println
+  def programNames = map (_.getDUTTestProgram.getTestProgramPlanName) dutTests
+  def simOutputs = map (singleSimOutput simName waves dut buildRoot) dutTests
```

Comment:
maybe consider mapping a function that returns the name -> output pair. since these two lists are get zipped together twice.